skyVegasTests/soundCheckGameClient.spec.ts

```diff
@@ -17,22 +23,23 @@ readGames().forEach((game) => {
     });
 
     test("Test sound toggle", async () => {
-      const soundToggleGameWindow = page.frameLocator("#root iframe").locator("i").nth(2)
+      const soundToggleIcon = page
+        .frameLocator("#root iframe")
+        .locator("i")
+        .nth(2);
       startEventListener(page, consoleMessages);
       await launchGame(page, game, consoleMessages);
-      if (messageExists(consoleMessages, ExpectedMessage.SOUND_CHECK_TOOL_BAR_ON)) {
-        await soundToggleGameWindow.click();
-        await validateConsoleMessages(
-          ExpectedMessage.SOUND_CHECK_TOOL_BAR_OFF,
-          consoleMessages
-        );
-      } else if (messageExists(consoleMessages, ExpectedMessage.SOUND_CHECK_TOOL_BAR_OFF)) {
-        await soundToggleGameWindow.click();
-        await validateConsoleMessages(
-          ExpectedMessage.SOUND_CHECK_TOOL_BAR_ON,
-          consoleMessages
-        );
-      }
+      await soundToggleIcon.click();
+      await soundToggleIcon.click();
```

Comment:
is it meant to be toggled twice here to turn it on initially?

Reply:
I think this is because the `SOUND_CHECK_GAME_ON` message isn't present when you first load up the game, so to get both messages ON and OFF we want to toggle the sound ON and OFF after we launch the game? That would make sense why we do it twice here

Reply:
We need to check that the sound toggle messages come up in the console. There is no need to check the initial status of the sound icon. Clicking it twice ensures that both on and off messages are pushed onto the console.